src/many.js

```diff
@@ -0,0 +1,33 @@
+// many.js
+export function setMany(entries, set) {
+	if (!Array.isArray(entries)) {
+		throw new TypeError("Entries must be an array");
+	}
+	entries.forEach((entry) => {
+		if (Array.isArray(entry)) {
+			const [key, value] = entry;
+			entry = { [key]: value };
+		}
+		const key = Object.keys(entry);
+		set(key[0], entry[key]);
+	});
+	return true;
+}
+
+export async function getMany(names, get) {
```

Comment:
for consistency, I think `getMany()` and `removeMany()` should still accept objects (as well as arrays), and it just use the keys -- ignoring the values -- from the object.